### ISSUE TITLE
Cluster Explorer search is not keyboard navigable

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#18984]
<img width="432" height="319" alt="Image" src="https://github.com/user-attachments/assets/73fc8b70-b695-4acc-8e77-951c0b8e5d96" />

When the 'Jump to' is focused, there is no focus ring - this is minor.

When focused, it shows the drop down of options - but you can't use up/down to navigate this drop down.
